src/aiokatcp/sensor.py

```diff
@@ -823,23 +838,37 @@ def update_aggregate(self, updated_sensor, reading, old_reading) -> Reading:
         """
         pass
 
+    def filter_aggregate(self, sensor: Sensor) -> bool:
+        """Check that a sensor isn't this aggregate sensor.
+
+        Users can override this function to exclude certain categories of
+        sensors, such as other aggregates, to prevent circular references.
+
+        Returns
+        -------
+        bool
```

Comment:
```suggestion
        include
```

Reply:
In the case of a single return, the numpy style guide suggests that this field should have the type:
https://numpydoc.readthedocs.io/en/latest/format.html#returns
Lower down it seems to suggest using descriptive names more for if there are multiple returns.

Reply:
You're right. I had mis-remembered.